projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -202,7 +216,7 @@
 		</p>
 	</Module>
 
-	<Module slug="lazy_images">
+	<Module slug="lazy_images" hideIfUnavailable={false}>
```

Comment:
Seems like we're adding an optional property to component that's only used in one component. Maybe it would be just simpler to wrap the content of the slot with a single `if` rather than changing the component?

Reply:
I was thinking in terms of using this for ISA as well. The latest design suggestion keeps the ISA module visible even in free. And, I think we'll need a toggle button to disable the fixes applied by ISA.